src/v8env.ts

```diff
@@ -56,33 +61,33 @@ export class V8Environment extends EventEmitter {
     })
   }
 
-  startCodeUpdater() {
+  startUpdater(){
     try {
-      if (runtimeConfig.env === 'development')
-        compiler.watch({}, this.updateV8Env.bind(this))
-      else
-        compiler.run(this.updateV8Env.bind(this))
+      console.log("Watching for changes:", v8dist)
+      let fsTimeout = false
+      fs.watch(v8dist, (eventType, fileName) => {
```

Comment:
Wouldn't it be nicer to use a programmatic webpack? Require the webpack v8 config (instead of defining it inline here) and run it like before. Instead of running webpack in a different process.

Reply:
Leaving the separate webpack because:
* Building v8env is not something anyone will ever do from the fly cli
* v8env should probably be a separate repo/build/project in the future